### PR TITLE
Update submodule with latest changes. Adapt spec to new changes.

### PR DESCRIPTION
**Pull request by mlehtima**
Silence some rpmlint warnings from spec file.

### CODE REVIEW

File: rpm/libhybris.spec [hunk 1]
```diff
@@ -31,7 +31,6 @@ Requires: %{name}-libhardware = %{version}-%{release}
 Requires(post): /sbin/ldconfig
 Requires(postun): /sbin/ldconfig
 Provides: libEGL
-Provides: libEGL.so.1
```
Comment (krnlyng) [inline rpm/libhybris.spec:34]: why this change?

Comment (sledges) [reply rpm/libhybris.spec:34]: With this change, on OBS package information doesn't write out "Provides: libEGL.so.1" twice :)
